frontend/src/services/applicationService.js

```diff
@@ -231,8 +231,8 @@ export default {
   },
   isCCOFCompleteFamilyV2(funding) {
     if (isEmpty(funding)) return false;
+    const licenceCategoryNumber = funding.familyLicenseType || funding.licenceCategoryNumber;
```

Comment:
Instead of adding a workaround for the two field names I think we should make the screens consistent to avoid issues down the road.